framework/src/modules/chain/rounds/rounds.js

```diff
@@ -375,11 +375,6 @@ class Rounds {
 		return library.delegates.validateBlockSlot(block);
 	}
 
-	// eslint-disable-next-line class-methods-use-this
-	generateDelegateList(round, source, tx) {
```

Comment:
I just noticed, the new `getRoundDelegates` method doesn't accept a `tx` object. Would this be a problem? Should we create an issue for it?

Reply:
I couldn't see any usage with tx object for previous implementation.

Reply:
If it was not used before I would leave without